problems/strings.js

```diff
@@ -3,23 +3,29 @@
  * @param {string} - str
  * @returns {string} - first char
  */
-function firstChar() {}
+const firstChar = (str) => {
+  return str.slice(0, 1);
```

Comment:
There's an easier way to do this than using slice. Think about how we get the first element of an array